apps/console/package.json

```
@@ -69,16 +71,16 @@
         "@wso2is/admin.groups.v1": "^2.27.117",
         "@wso2is/admin.home.v1": "^1.7.2",
         "@wso2is/admin.identity-verification-providers.v1": "^2.27.203",
+        "@wso2is/admin.impersonation.v1": "^1.6.203",
```

Let's use the workspace protocol for the updated packages, so that you wouldn't have to deal with merge conflicts.

```suggestion
        "@wso2is/admin.impersonation.v1": "workspace:^",
```